src/gatsby/utils/packageRegistry.ts

```diff
@@ -45,28 +43,16 @@ export default class PackageRegistry {
   };
 
   getData = async (name: string) => {
-    if (!this.cache[name]) {
-      console.info(`Fetching release registry for ${name}`);
-      try {
-        const result = await axios({
-          url: `https://release-registry.services.sentry.io/sdks/${name}/latest`,
-        });
-        this.cache[name] = result.data;
-      } catch (err) {
-        console.error(`Unable to fetch registry for ${name}: ${err.message}`);
-        this.cache[name] = {};
-      }
-    }
-
-    return this.cache[name];
+    await this.getList();
+    return this.indexCache[name]
   };
 
-  version = async (name, defaultValue: string = "") => {
+  version = async (name: string, defaultValue: string = "") => {
     const data = (await this.getData(name)) as VersionData;
-    return data.version || defaultValue;
+    return data && data.version || defaultValue;
```

Comment:
Are you TS yet?

```suggestion
    return data?.version || defaultValue;
```

Reply:
i'm in fact not ts yet, doing this breaks

i think we're running an old ts

Reply:
or maybe I don't know how to ts 😁

Reply:
no I tried before, and all the stuff we can do in sentry frontend doesn't work here